system/CLI/CLI.php

```diff
@@ -150,21 +150,16 @@ public static function init()
             // much more bash-like.
             // http://www.php.net/manual/en/readline.installation.php
             static::$readline_support = extension_loaded('readline');
-
```

Comment:
Why do you remove this empty line?

Reply:
The rector did it. I don't know why he doesn't like it the way it was.

Reply:
@kenjis reverted rector line changes.